bin/studio.py

```diff
@@ -0,0 +1,276 @@
+from ensurepip import version
+import argparse
```

Comment:
I prefer `bin/studio.py` as it's an cli interface to Studio, `cli/cli.py` really does not tell anything to me.
I actually have a rather different issue with the current setup. The 
``` bin/run bin/studio.py --debug --config config-some-subdomain.json download_media --media_id 49```

feels really awkward to me, because the bin/run thingie. What could work is to have a shell script living in the `bin` directory (yes, another wrapper), calling the python script. In that case, we can move `bin/studio.py` wherever we want.

Reply:
we agreed to move `bin/studio.py` to `cli/cli.py` and have `bin/cli` wrapper script to execute `bin/run cli/cli.py <parameters>"